tests/test_flow/test_protocols.py

```diff
@@ -372,6 +372,26 @@ def np_array_high_d():
     assert(np.array_equal(arr, desired))
 
 
+def test_yaml_protocol_list(builder):
+    x = ['red', 'blue']
```

Comment:
nit: Is there a better name we can use here than `x`?

Reply:
how about getting rid of `x` entirely?  it means duplicating `['red', 'blue']` (i.e. in the entity and the assertion), but seems very minor

Reply:
Yeah, I think that makes sense here.